scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,531 @@
+      path: "/etc/kat/rocky.conf"
+      regexp: "DJANGO_ALLOWED_HOSTS=.*"
+      line:  "DJANGO_ALLOWED_HOSTS=\"{{ ansible_default_ipv4.address }}\""
+
+  - name: Configure hostname in Rocky. Currently we configure IP address. Please add FQDN later to  DJANGO_CSRF_TRUSTED_ORIGINS setting in /etc/kat/rocky.conf !
+    ansible.builtin.lineinfile:
+      path: "/etc/kat/rocky.conf"
+      regexp: "DJANGO_CSRF_TRUSTED_ORIGINS=.*"
+      line:  "DJANGO_CSRF_TRUSTED_ORIGINS=\"{{ django_csrf }}\""
+
+  # Enable debugging on rocky
+  # default: DEBUG="False"
+  - name: Enable debugging on Rocky.
+    ansible.builtin.lineinfile:
+      path: /etc/kat/rocky.conf
+      regexp: "DEBUG=.*"
+      line:  "DEBUG=\"True\""
+
+  # Initialize the Bytes database:
+  # sudo -u kat update-bytes-db
+  - name: initialize Bytes database
+    become: true
+    become_user: kat
+    ansible.builtin.shell:
+      cmd: update-bytes-db
+
+  # Initialize the Mula database:
+  # sudo -u kat update-mula-db
+  - name: initialize Mula database
+    become: true
+    become_user: kat
+    ansible.builtin.shell:
+      cmd: update-mula-db
+
+  # Initialize the database using the update-katalogus-db tool:
+  #
+  # sudo -u kat update-katalogus-db
+  - name: initialize katalogus database
+    become: true
+    become_user: kat
+    ansible.builtin.shell:
+      cmd: update-katalogus-db
+
+  # Restart KAT
+  # After finishing these steps, you should restart KAT to load the new configuration:
+  # sudo systemctl restart kat-rocky kat-mula kat-bytes kat-boefjes kat-normalizers kat-katalogus kat-keiko kat-octopoes kat-octopoes-worker
+  # Start KAT on system boot
+  # To start KAT when the system boots, enable all KAT services:
+  # sudo systemctl enable kat-rocky kat-mula kat-bytes kat-boefjes kat-normalizers kat-katalogus kat-keiko kat-octopoes kat-octopoes-worker
+  - name: Start  kat-rocky kat-mula kat-bytes kat-boefjes kat-normalizers kat-katalogus kat-keiko kat-octopoes kat-octopoes-worker
+    ansible.builtin.systemd:
+      name: "{{ item }}"
+      state: restarted
+      enabled: true
+    loop:
+      - kat-mula
+      - kat-bytes
+      - kat-boefjes
+      - kat-normalizers
+      - kat-katalogus
+      - kat-keiko
+      - kat-octopoes
+      - kat-octopoes-worker
+      - kat-rocky
+
+  # Start using OpenKAT
+  #
+  # By default OpenKAT will be accessible in your browser through https://<server IP>:8443 (http://<server IP>:8000 for docker based installs). There, Rocky will take you through the steps of setting up your account and running your first boefjes.
+  # Upgrading OpenKAT
+  #
+
+  - name: Important post-install message
+    ansible.builtin.pause:
+      seconds: 1
+      prompt: |
+        ********************************************************************************************************
+        * After a fresh installation of OpenKAT you should be able to start using it.
+        "* Visit https://{{ ansible_default_ipv4.address }}{{':'}}8443."
```

Comment:
We have moved to Granian on port http at 8000.

Reply:
Of course it has. This PR is getting old. Trying to keep up with a complete OpenKAT team is like playing wackamole. Also, stuff changes between releases.
Kind request to accept this PR so I can start working on the update.